apps/omg/test/omg/state/core_test.exs

```diff
@@ -924,6 +1174,7 @@ defmodule OMG.State.CoreTest do
   # used to check the invariants in form_block
   # use this throughout this test module instead of Core.form_block
   defp form_block_check(state) do
+    state = Core.claim_fees(state)
```

Comment:
I think it's better to split it out and explicitly call `claim_fees` where applicable / expected.